### PR TITLE
Implemented cancle appointment logic on backend and unit tests

### PR DESCRIPTION
**Pull request by aca24**

### CODE REVIEW

File: MQuince/MQuince.Services.Implementation/AppointmentService.cs
Comment (dusancina98) [inline MQuince/MQuince.Services.Implementation/AppointmentService.cs]: Method return type sholud be boolean to ensure if Appointment is canceled or not.
You can get current date with DateTime.Now. Method parameter 'today' is not necessary

File: MQuince/MQuince.Services.Implementation/AppointmentService.cs
Comment (dusancina98) [inline MQuince/MQuince.Services.Implementation/AppointmentService.cs]: Method GetById for Appointment is already implemented.

File: MQuince/MQuince.Services.Tests/UnitTests/AppointmentTests.cs
Comment (dusancina98) [inline MQuince/MQuince.Services.Tests/UnitTests/AppointmentTests.cs]: Do you create Appointment Stub Repository or Work Time Stub Repository?

File: MQuince/MQuince.Services.Tests/UnitTests/AppointmentTests.cs
Comment (dusancina98) [inline MQuince/MQuince.Services.Tests/UnitTests/AppointmentTests.cs]: From Test method name we don't see what is input and expected result

**Review by dusancina98 — CHANGES_REQUESTED** [review]

**Review by zindovicstefan — CHANGES_REQUESTED** [review]
Following requested changes are not implemented:
Method GetById for Appointment is already implemented. Use it instead getAppointment
Method return type sholud be boolean to ensure if Appointment is canceled or not.

**Review by zindovicstefan — CHANGES_REQUESTED** [review]
‎As we have discussed earlier this branch in this state introduces breaking change. Because appointment entity is modified to support logical deletion by adding additional boolean property, repository methods don't work properly anymore. Each method which fetches appointments from database should be modified to check if appointment is logically deleted.

**Review by zindovicstefan — CHANGES_REQUESTED** [review]
Update feature branch with develop branch, Integration and Unit Tests need to be in correct namespace so tests can run in CI pipeline. CI pipeline failed so this changes can not be approved. Build fails on add-migration locally when branch is updated with develop. With @dusancina98 I tried multiple times to add migration each times unsuccessfully. We can't approve this changes day before sprint review, high possibility to introduce problems with continuous delivery to heroku.

**Comment by zindovicstefan** [comment]
This branch has conflicts that must be resolved before merging to develop. Update branch with changes from develop before PR. Conflicting file are listed under CI pipeline checks on this page.

**Pull request closed without merging**
